src/OutdatedLibraries/OutdatedFileToTable/LibraryOutdated.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\OutdatedLibraries\OutdatedFileToTable;
+
+
+class LibraryOutdated
```

Comment:
```suggestion
class Library
```
On pourrait utiliser la class même si les library ne sont pas outdated donc niveau nom juste Library c'est bien